app/d3plus.css

```diff
@@ -7,35 +7,77 @@
 
 .d3plus-tooltip {
   box-shadow: rgba(0, 0, 0, 0.25) 0px 1px 3px;
-  max-width: 225px;
-  min-width: 150px;
+  max-width: 224px;
+  min-width: 224px;
```

Comment:
Shouldn't this just be `width: 224px;`?

If there is a conflicting default min or max width, just set that to `auto`.